### DISCUSSION TITLE
Question Regarding Font Features in ScribeOCR

### DISCUSSION THREAD

**Discussion by ashokparmar29** [scribeocr/scribeocr#58] [Q&A]
First of all, thank you for the great work. It’s an impressive tool, and I appreciate all the effort put into its development.

I am trying to understand how font features are replicated in ScribeOCR. I have read the FAQs but still have some confusion. Is ABBYY used to obtain the font features, or does Tesseract return the font information which is then used to replicate the font features in the output?

Any clarification on this would be greatly appreciated. Thank you in advance for your help.

**Comment by rmast** [comment]
I guess font features like serif/sans italic/straight are recognized and then a basic font is matched.

https://github.com/scribeocr/scribeocr/blob/d015ef9afff63900d59b08291891fcbdc10f8c91/js/utils/miscUtils.js#L321

**Comment by Balearica** [comment]
Closing as duplicate of #52, feel free to discuss font optimization there.  I believe these questions are answered by [this comment](https://github.com/scribeocr/scribeocr/discussions/52#discussioncomment-10278514).  If not, feel free to leave a comment in the other discussion.
